doc/command_ref.rst

```diff
@@ -442,8 +442,9 @@ Groups are virtual collections of packages. DNF keeps track of groups that the u
 
 ``dnf [options] group list <group-spec>...``
     List all matching groups, either among installed or available groups. If
-    nothing is specified list all known groups. Options ``installed`` and ``available`` narrows down the requested list.
+    nothing is specified list all known groups. Options ``--installed`` and ``--available`` narrows down the requested list.
     Records are ordered by `display_order` tag defined in comps.xml file.
+    Provides a list of all hidden groups by using option ``--hidden`` instead of group name.
```

Comment:
```instead of group name``` - with option it doesn't make sense.   I think it has to be removed or rephrased.